src/node/utils/ExportHtml.js

```diff
@@ -315,8 +315,11 @@ async function getHTMLFromAtext(pad, atext, authorColors) {
       }
       await hooks.aCallAll('getLineHTMLForExport', context);
       // To create list parent elements
-      if ((!prevLine || prevLine.listLevel !== line.listLevel) || (prevLine && line.listTypeName !== prevLine.listTypeName)) {
-        const exists = _.find(openLists, (item) => (item.level === line.listLevel && item.type === line.listTypeName));
+      if ((!prevLine || prevLine.listLevel !== line.listLevel) ||
+        (prevLine && line.listTypeName !== prevLine.listTypeName)) {
+        const exists = _.find(openLists, (item) => (
```

Comment:
Nit: It's more readable if you indent the continuation lines 4 or more spaces more than the starting line's indentation:

```suggestion
      if ((!prevLine || prevLine.listLevel !== line.listLevel) ||
          (prevLine && line.listTypeName !== prevLine.listTypeName)) {
        const exists = _.find(openLists, (item) => (
```